<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="open_File">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>open File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>951</width>
     <height>551</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camerabtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>camera on</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recordbtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Rec Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="capturebtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>capture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="videobtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RGB">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>130</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HSV">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>130</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>HSV</string>
   </property>
  </widget>
  <widget class="QSlider" name="R_or_H">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="G_or_S">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="B_or_V">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>R_or_H</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>G_or_S</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>B_or_V</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
